src/common/fs/fs.cpp

```diff
@@ -609,13 +609,28 @@ u64 GetSize(const fs::path& path) {
 
     const auto file_size = fs::file_size(path, ec);
 
+#ifdef ANDROID
```

Comment:
Duplicated control flow. Try this:
```suggestion
#ifdef ANDROID
    if (Android::IsContentUri(path)) {
        return Android::GetSize(path);
    }
#endif

    const auto file_size = fs::file_size(path, ec);
```

Reply:
Hm, it should actually be impossible to get a normal path instead of a uri so I'll simplify based on that

Reply:
Oops, no that should only be the case with games directories and I should know that this could be used for anything within the private data directory that can get a normal path.